Neos.ContentRepository.Core/Classes/CommandHandlingDependencies.php

```diff
@@ -82,8 +85,11 @@ public function getContentGraph(WorkspaceName $workspaceName): ContentGraphInter
         if (isset($this->overriddenContentGraphInstances[$workspaceName->value])) {
             return $this->overriddenContentGraphInstances[$workspaceName->value];
         }
-
-        return $this->contentRepository->getContentGraph($workspaceName);
+        $workspace = $this->contentGraphReadModel->findWorkspaceByName($workspaceName);
+        if ($workspace === null) {
+            throw WorkspaceDoesNotExist::butWasSupposedTo($workspaceName);
+        }
```

Comment:
With the [CR Privilges PR](https://github.com/neos/neos-development-collection/pull/5298) the evaluation of permissions is centralized to the [ContentRepository](https://github.com/neos/neos-development-collection/blob/bf57af6d351550860be19f28b8318889fc9d2836/Neos.ContentRepository.Core/Classes/ContentRepository.php#L250-L268)
I _think_ it's fine (or even desired) that they are _not_ evaluated here, but I'm not 100% sure